website/pages/en/querying/querying-from-an-application.mdx

```diff
@@ -2,49 +2,61 @@
 title: Querying from an Application
 ---
 
-Once a subgraph is deployed to Subgraph Studio or to Graph Explorer, you will be given the endpoint for your GraphQL API that should look something like this:
+Learn how to query The Graph from your application.
 
-**Subgraph Studio (testing endpoint)**
+## Getting GraphQL Endpoint
 
-```sh
-Queries (HTTP)
+Once a subgraph is deployed to [Subgraph Studio](https://thegraph.com/studio/) or [Graph Explorer](https://thegraph.com/explorer), you will be given the endpoint for your GraphQL API.
+
+### Subgraph Studio
+
+Your GraphQL API in Studio should look something like this:
+
+```
 https://api.studio.thegraph.com/query/<ID>/<SUBGRAPH_NAME>/<VERSION>
 ```
 
-**Graph Explorer**
+> Note: Remember that Subgraph Studio is a testing playground for subgraphs.
```

Comment:
@idalithb, please ask someone else's opinion, but this is not quite what I meant with my feedback above. In fact, I am pretty sure Studio will show the production endpoint when a subgraph is published to the network, so I think it's important to distinguish clearly between "development" and "production" endpoints.